db/migrate/20200312144232_rename_value_on_pr_state.rb

```diff
@@ -1,13 +0,0 @@
-class RenameValueOnPrState < ActiveRecord::Migration[6.0]
```

Comment:
I would avoid modifying/deleting migrations that were already run in an environment. In this case maybe it's better to create a new migration to rename the column again.

Reply:
Ok! Thought of not repeating code